docs/micro-frontends-in-open-edx.rst

```diff
@@ -0,0 +1,95 @@
+#######################################
+Micro-frontend applications in Open edX
+#######################################
+
+The purpose of this document is to provide an overview of how micro-frontend applications (MFEs) are developed, configured, and deployed with the Open edX ecosystem. Open edX MFEs are small React applications that can be built and deployed independently.
+
+MFEs are static site generators. They are supplied configuration, then run to build HTML, CSS, and Javascript. The static assets are then ready then be moved to a location and served to the user.
+
+All MFEs offer the following commands:
```

Comment:
We may want to say that you need to have a node 12 environment available on your development machine.  We don't have to say _how_ to install it, but just give people some pointers to external resources and specific version numbers (i.e., node 12 and the latest version of npm 6)